cmd/single-ocsp/main.go

```diff
@@ -84,6 +85,33 @@ func readFiles(issuerFileName, responderFileName, targetFileName, pkcs11FileName
 	return
 }
 
+func signerValidForResp(issuer, responder *x509.Certificate, thisUpdate, nextUpdate time.Time) error {
+	if !issuer.Equal(responder) {
```

Comment:
With this update, the check on thisUpdate / nextUpdate being within the validity interval of the signing certificate doesn't when signing directly from an issuer. Is there a reason for that?